src/validateURL.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable max-len */
+const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const errors = {
```

Comment:
Use more descriptive names.
```suggestion
const errorsMessages = {
```